src/parser/ast_common.h

```diff
@@ -27,6 +27,7 @@ typedef struct {
 	char *label;			// Label of entity.
 	Vector *properties;		// Array of attributes.
 	AST_GraphEntityType t;	// Type of entity.
+	bool anonymous;			// Entity isn't referenced.
```

Comment:
I like this! We can probably get away from "anon_*" naming entirely with a bit of work, I think, which would be very nice.